packages/@react-aria/color/src/useColorSwatch.ts

```diff
@@ -21,7 +21,7 @@ import {useLocale, useLocalizedStringFormatter} from '@react-aria/i18n';
 
 export interface AriaColorSwatchProps extends AriaLabelingProps, DOMProps {
   /** The color value to display in the swatch. */
-  color?: string | Color | null,
+  color?: string | Color,
```

Comment:
Technically this can still be undefined so I guess we'd want it to be non-optional?

Reply:
I started with that, but isn't there the case where RAC user's just want it to use the value from context?